gcc/rust/expand/rust-macro-builtins.cc

```diff
@@ -453,8 +453,12 @@ MacroBuiltin::include_bytes_handler (Location invoc_locus,
   std::string target_filename
     = source_relative_path (lit_expr->as_string (), invoc_locus);
 
-  std::vector<uint8_t> bytes
-    = load_file_bytes (invoc_locus, target_filename.c_str ());
+  auto maybe_bytes = load_file_bytes (invoc_locus, target_filename.c_str ());
+
+  if (!maybe_bytes.has_value ())
+    return AST::Fragment::create_error ();
+
+  std::vector<uint8_t> bytes = maybe_bytes.value ();
```

Comment:
it's going to be really cool if this function can return a `tl::optional<AST::Fragment>` because then we'll be able to just `map` it, like *proper functional programmers*!

Reply:
hmm, I'll have to read a bit more here, as I'm not sure to follow what changes this would allow